src/health-leaderboard/cloudwatch/health-leaderboard-cfn-template.js

```diff
@@ -0,0 +1,43 @@
+const cf = require('@mapbox/cloudfriend')
+
+const Resources = {
+  HealthLeaderboardLambda: {
+    Type: 'AWS::Lambda::Function',
+    Properties: {
+      FunctionName: 'health-leaderboard',
+      Handler: 'health-leaderboard-lambda.handler',
+      Role: cf.sub(
+        'arn:aws:iam::${AWS::AccountId}:role/service-role/test-slack-router-role-4hs4dpro'
```

Comment:
`cf.arn` again

Reply:
@ramyaragupathy 
Thank for flagging.

Once I get confirmation that we'll create a separate role for each Lambda then I'll replace this to reference their own respective role ARN instead. :)